shub/cancel.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import absolute_import
+
+import click
+
+from scrapinghub import ScrapinghubClient, ScrapinghubAPIError
+from scrapinghub.client.utils import parse_job_key
+
+from shub.config import get_target_conf
+from shub.exceptions import (
+    ShubException,
+    BadParameterException,
+    SubcommandException,
+    RemoteErrorException,
+)
+
+
+HELP = """
+Cancel multiple jobs from Scrapy Cloud.
+
+The cancel command expects the project ID followed by the pair
+containing the spider ID and Job ID:
+
+    shub cancel 12345 1/1 1/2 1/3
+
+If the project ID is not defined it is going to use the default
+project (as defined in scrapinghub.yml):
+
+    shub cancel 1/1 1/2 1/3
+"""
+
+
+SHORT_HELP = "Cancel multiple jobs from Scrapy Cloud"
+
+
+@click.command(help=HELP, short_help=SHORT_HELP)
+@click.argument("project_id")
+@click.argument("keys", nargs=-1)
+def cli(project_id, keys):
+    # Project ID may be defined or not
+    # When user doesn't speficy then the project_id
+    # is just another key
+    if "/" in project_id:
+        keys = (project_id,) + keys
+        target = "default"
+    else:
+        target = project_id
+
+    targetconf = get_target_conf(target)
+    project_id = targetconf.project_id
+    client = _get_client(targetconf)
+
+    try:
+        project = client.get_project(project_id)
+    except ScrapinghubAPIError as err:
+        raise RemoteErrorException(str(err))
+
+    job_keys = [validate_job_key(project_id, key) for key in keys]
+
+    try:
+        output = project.jobs.cancel(
+            keys=[str(job) for job in job_keys]
+        )
+    except ValueError as err:
+        raise ShubException(str(err))
+
+    click.echo(output)
```

Comment:
Wonder if we should wrap the output with human readable message, like "N job(s) were successfully cancelled.", or say that no jobs were cancelled.

Reply:
Yeah, I thought about that, but in the same time there are other commands like `shub delpoy`.. 
![image](https://user-images.githubusercontent.com/163367/62711689-e00fa000-b9cf-11e9-8b06-1b9c0ad134f7.png)
But of course we could change it to a human readable message :)

Reply:
You're right, we could keep it as-is for now, I think the output should be clear enough :+1: